kernel/kernel-impl/src/main/java/org/sakaiproject/tasks/impl/TaskServiceImpl.java

```diff
@@ -187,8 +219,29 @@ public void completeUserTaskByReference(String reference, List<String> userIds)
 
     @Transactional
     public void removeTask(Task task) {
-
+        taskAssignedRepository.deleteByTask(task);
         userTaskRepository.deleteByTask(task);
         taskRepository.delete(task);
     }
+    
+    @Transactional
+    public void assignTask(Task task, String type, String objectId) {
+        assignTask(task, type, Arrays.asList(objectId));
+    }
+    
+    @Transactional
+    public void assignTask(Task task, String type, List<String> objectIds) {
+        for (String objectId : objectIds) {
```

Comment:
Please leave a blank line after the method name in these classes. The rest of the code does that as it makes things, imo, easier to read. Only on multiline method bodies though, I don't leave a blank when the method is just one line. That would be crazy.

Reply:
Same as above. This is a cosmetic decission, and I don't agree with that.

Reply:
cosmetics are important in code - they can make a big difference to legibility. I think it's easier to read, and the rest of the service has them in since I wrote it. If you really want to make a stand and not have them in your methods then I'll have to accept that, but don't modify existing code, please.